### ISSUE TITLE
Deno Support

### ISSUE THREAD

**Issue by abdulrahman1s** [eleith/emailjs#316]
It would be cool if we could use this library in Deno. :smile:

**Comment by zackschuster** [comment]
yeah, that'd be neat! have you tried running it in [compatibility mode](https://deno.land/manual/node/compatibility_mode)?

**Comment by zackschuster** [comment]
ah, bummer. i love the idea, though. how would we implement it? i thought of using an [https://deno.land/manual/linking_to_external_code/import_maps](import map) to point the node built-ins to a separate file that re-exports deno built-ins, but i'm not sure if that's the right approach.

**Comment by abdulrahman1s** [comment]
Deno has built-in modules to do the job such as [`tls`](https://doc.deno.land/deno/stable/~/Deno.startTls)
But it'll be hard to maintain node and deno at the same time..
Maybe if we could create adapters to make things easier to automate the deno version would be great.

**Comment by lucsoft** [comment]
Hello Denoers! I'm advancing deno support in https://github.com/lucsoft/emailjs with better typing and all that fancy stuff. Currently it requires node api but im hoping to go full Web Standards + Deno TLS/Socket sutff
